packages/expo-sms/ios/EXSMS/EXSMSModule.m

```diff
@@ -20,84 +18,77 @@ @implementation EXSMSModule
-  _utils = [moduleRegistry getModuleImplementingProtocol:@protocol(UMUtilitiesInterface)];
+    _utils = [moduleRegistry getModuleImplementingProtocol:@protocol(UMUtilitiesInterface)];
 }
 
 UM_EXPORT_METHOD_AS(isAvailableAsync,
                     isAvailable:(UMPromiseResolveBlock)resolve
-                       rejecter:(UMPromiseRejectBlock)reject)
+                    rejecter:(UMPromiseRejectBlock)reject)
 {
-  dispatch_async(dispatch_get_main_queue(), ^{
-    BOOL canOpenURL = [[UIApplication sharedApplication] canOpenURL:[NSURL URLWithString:@"sms:"]];
-    resolve(@(canOpenURL));
-  });
+    resolve(@([MFMessageComposeViewController canSendText]));
 }
 
 UM_EXPORT_METHOD_AS(sendSMSAsync,
                     sendSMS:(NSArray<NSString *> *)addresses
                     message:(NSString *)message
-                   resolver:(UMPromiseResolveBlock)resolve
-                   rejecter:(UMPromiseRejectBlock)reject)
+                    resolver:(UMPromiseResolveBlock)resolve
+                    rejecter:(UMPromiseRejectBlock)reject)
 {
-  if (![MFMessageComposeViewController canSendText]) {
-    reject(@"E_SMS_UNAVAILABLE", @"SMS service not available", nil);
-    return;
-  }
-
-  if (_resolve != nil || _reject != nil) {
-    reject(@"E_SMS_SENDING_IN_PROGRESS", @"Different SMS sending in progress. Await the old request and then try again.", nil);
-    return;
-  }
-
-  _resolve = resolve;
-  _reject = reject;
-  
-  MFMessageComposeViewController *messageComposeViewController = [[MFMessageComposeViewController alloc] init];
-  messageComposeViewController.messageComposeDelegate = self;
-  messageComposeViewController.recipients = addresses;
-  messageComposeViewController.body = message;
-
-  UM_WEAKIFY(self);
-  [UMUtilities performSynchronouslyOnMainThread:^{
-    UM_ENSURE_STRONGIFY(self);
-    [self.utils.currentViewController presentViewController:messageComposeViewController animated:YES completion:nil];
-  }];
+    if (![MFMessageComposeViewController canSendText]) {
+        reject(@"E_SMS_UNAVAILABLE", @"SMS service not available", nil);
+        return;
+    }
+    
+    if (_resolve != nil || _reject != nil) {
+        reject(@"E_SMS_SENDING_IN_PROGRESS", @"Different SMS sending in progress. Await the old request and then try again.", nil);
+        return;
+    }
+    
+    _resolve = resolve;
+    _reject = reject;
+    
+    MFMessageComposeViewController *messageComposeViewController = [[MFMessageComposeViewController alloc] init];
+    messageComposeViewController.messageComposeDelegate = self;
+    messageComposeViewController.recipients = addresses;
+    messageComposeViewController.body = message;
+    
+    UM_WEAKIFY(self);
+    [UMUtilities performSynchronouslyOnMainThread:^{
+        UM_ENSURE_STRONGIFY(self);
+        [self.utils.currentViewController presentViewController:messageComposeViewController animated:YES completion:nil];
+    }];
```

Comment:
Our whole codebase is configured to have 2-space indent. Are you changing this intentionally to 4-space indentation?

Reply:
Just used the auto-format